antarest/storage/repository/filesystem/config/model.py

```diff
@@ -29,7 +29,7 @@ class Area(DTO):
     def __init__(
         self,
         links: Dict[str, Link],
-        thermals: List[str],
+        thermals: Dict[str, Any],
```

Comment:
I prefere to expand domain with new `Thermal` object. Like there are for `Link`. Today, there is only `enable` attribute in `Thermal` but it will be more easy to upgrade if we need to add other attributes.

Reply:
Indeed this would be cleaner